app/views/users/sessions.py

```diff
@@ -141,6 +146,77 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    main = []
+    data = ["Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status", \
```

Comment:
@SaptakS 
I am appending it cause when writing it with **writerow** function , the main list will contain sublists , each sublists comprising of each row in the csv file .

That's why such an arrangement.